plugins/hooks.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2023 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package plugins
+
+import (
+	"github.com/open-policy-agent/opa/hooks"
+)
+
+func WithHooks(hs hooks.Hooks) func(*Manager) {
+	return func(m *Manager) {
+		m.hooks = hs
+	}
+}
```

Comment:
Nit: What's the reason to create a new file for this v/s adding it in `plugins/plugins.go`?